bilby/gw/prior.py

```diff
@@ -1138,7 +1138,7 @@ def to_file(self, outdir, label):
     @staticmethod
     def from_envelope_file(envelope_file, minimum_frequency,
                            maximum_frequency, n_nodes, label,
-                           boundary="reflective"):
+                           boundary="reflective", correction=None):
```

Comment:
Agree with `correction` -> `correction_type`, but also I think the default in the signature should be `correction_type='data'` and there should just be a message